tests/core/test_save_files.py

```diff
@@ -64,3 +64,19 @@ def test_save_file():
         in result.stderr.decode()
     )
     assert result.returncode == 1
+
+
+def test_save_cloud_file():
+    # should be no key for cloud paths
+    result = subprocess.run(
+        "lamin save s3://lamindb-ci/lndb-storage/testfile.hdf5 --key filekey.h5ad",
```

Comment:
Wait, this file should long have been deleted.

We're using it multiple places and we shouldn't add more usage of it.

Instead, we should re-use proper file paths. I think we have, e.g., a file that's called `mini.csv` in a sensible location.

The way I do it is go through the existing tests and then pick correct paths.

The most hurtful usage is here:

<img width="912" alt="image" src="https://github.com/user-attachments/assets/08fcaa3b-839c-475b-97b5-5c109b7ea01c" />

It looks random and unprofessional.

Reply:
I see the PR is now already merged and that's not a problem. Canonicalizing test data can be another PR.

Reply:
Ah, i see, just took some file.